thirdeye-notification/src/main/java/ai/startree/thirdeye/notification/NotificationEventsBuilder.java

```diff
@@ -64,7 +66,13 @@ public NotificationEventsBuilder(final EventManager eventDao,
   /**
    * TODO cyril - logic is duplicated (and not iso) with what RcaRelatedService#getRelatedEvents provides
    */
-  public List<EventApi> getRelatedEvents(final Collection<AnomalyDTO> anomalies) {
+  public List<EventApi> getRelatedEvents(final Collection<AnomalyDTO> anomalies,
+      SubscriptionGroupDTO subscriptionGroup) {
+    final String currentNamespace = subscriptionGroup.namespace();
```

Comment:
I think the namespace can be obtained from an anomaly

if (anomalies.isEmpty()) {
  return Collections.emptyList()
}
final String namespace = anomalies.get(0).namespace;

Reply:
makes sense, we don't need to pass subscription group just for this